src/components/common/navigation/navigation.js

```diff
@@ -47,22 +51,31 @@ export default class Navigation extends Component {
     }
   }
 
-  getNavAnchorLink = item => (
-    <AnchorLink href={`#${item.toLowerCase()}`} onClick={this.closeMobileMenu}>
-      {item}
-    </AnchorLink>
-  )
+  getNavLink = ({ name, pathname, hash }) => {
+    return window.location.pathname === pathname && hash ? (
+      <AnchorLink href={hash} onClick={this.closeMobileMenu}>
+        {name}
+      </AnchorLink>
+    ) : (
+      <Link
+        to={`${hash ? pathname + hash : pathname}`}
+        onClick={this.closeMobileMenu}
+      >
+        {name}
+      </Link>
+    )
+  }
```

Comment:
If current `window.location.pathname` matches the link's pathname, use an `AnchorLink` for smooth-scroll behavior. If current `window.location.pathname` does not match the link's pathname use a gatsby Link.

Reply:
Thanks for working on this. I see the build is currently failing with 
> error #95312 ReferenceError: window is not defined
4:04:43 PM: "window" is not available during server-side rendering. 

You can resolve this by adding additional logic to check if `typeof window !== 'undefined'` when the site is built outside of the browser context

Reply:
Thanks @M0nica ! Will update